components/islands/PullRequestCard.vue

```diff
@@ -47,6 +49,6 @@ defineProps<{ pullRequests: String }>()
       </defs>
     </svg>
   </div>
-  <span class="text-5xl font-medium">{{ pullRequests }}</span>
-  <span class="text-2xl">Merged PRs</span>
+  <span class="text-5xl font-medium">{{ format(pullRequests) }}</span>
+  <span class="text-2xl">{{ pullRequests === 1 ? 'Merged PR' : 'Merged PRs' }}</span>
```

Comment:
nit
```suggestion
  <span class="text-2xl">Merged {{ pullRequests === 1 ? 'PR' : 'PRs' }}</span>
```

Reply:
I have [changed it](https://github.com/nuxt/nuxters/compare/17fe8c5991ced83030dbd6d2f97226d5ef0547b3..06d56a83fc29e954642ebd3bbcb0fec37b5d9b9c). I had it the previous way to lessen line changes if i18n were ever added to this project. (The entire string "Merged PR" / "Merged PRs" would probably be localized instead of "Merged" and "PR"/"PRs" separately because other non-English languages might not make the phrase plural by just appending an "s".)